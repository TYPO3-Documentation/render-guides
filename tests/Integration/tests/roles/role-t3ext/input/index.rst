==============
Document Title
==============

As an example, you may want to take a look at extension
:t3ext:`fal_securedownload` or :t3ext:`EXT:news`, :t3ext:`news <EXT:news>`.
