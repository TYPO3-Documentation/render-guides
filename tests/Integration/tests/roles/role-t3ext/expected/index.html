<!-- content start -->
                <section class="section" id="document-title">
    <h1>Document Title</h1>

            <p>As an example, you may want to take a look at extension
<a class="reference external" href="https://extensions.typo3.org/extension/fal_securedownload" title="EXT:fal_securedownload in the TER (TYPO3 extension repository)">EXT:fal_securedownload</a>.</p>
    </section>

        <!-- content end -->
